.github/ISSUE_TEMPLATE/New_Currency_Request.yml

```diff
@@ -0,0 +1,35 @@
+name: New Currency Request
+title: "[Currency_Req]: "
+description: "Request to add currency for any new package with specific version on ppc64le"
+labels: "new currency"
+projects: "ppc64le/1"
+assignees: "ICH"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Github is reserved for bug reports and feature requests; it is
+        not the place for general questions. 
+        Please fill in the following details to request Currency for a package:
```

Comment:
Same comment as previous file...

Reply:
done.
`Please complete the following fields to request currency for new package.`